torchgeo/models/swin.py

```diff
@@ -16,14 +16,48 @@
 __all__ = ["Swin_V2_B_Weights"]
 
 
+# Custom transform for Sentinel-2 multispectral model inputs; Uses 9 bands.
+class MSS2Transform(torch.nn.Module):
```

Comment:
Converting this one to kornia might be more complicated. Let me draft something up

Reply:
This should do the trick:

```python
_std = torch.tensor([255.0, 255.0, 255.0, 8160.0, 8160.0, 8160.0, 8160.0, 8160.0, 8160.0)] # not sure if there are additional bands between the first 3 and last 6, but can add 1.0 to not modify them
_mean = torch.zeros_like(_std)
_sentinel2_ms_satlas_transforms = AugmentationSequential(
   K.Normalize(mean=_mean, std=_std),
   Lambda(lambda x: torch.clamp(x, min=0.0, max=1.0))
   data_keys=["image"]
)